### PR TITLE
Mathable

### LINKED ISSUE #116: Mathable module

### PR DESCRIPTION
**Pull request by sarynm12**
-Added average method to mathable module
-Updates in stat tracker 
-We can also think about using mathable in other classes, but wanted to push this first. Helpable seems unable to require another module

### CODE REVIEW

File: lib/readable.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+module Readable
+
+  def self.create(path, collection)
+    CSV.foreach(path, headers: true, header_converters: :symbol) do |row|
+      collection << self.new(row.to_h)
+    end
+  end
+
+end
```
Comment (AngelaGuardia) [inline lib/readable.rb]: I don't think we'll need this module since we created the superclass DataSet

File: test/test_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+require 'simplecov'
+SimpleCov.start
```
Comment (AngelaGuardia) [inline test/test_helper.rb:2]: I think this was part of changing the file name for helper_test.rb to test_helper.rb

test_helper should include:
require 'simplecov'
SimpleCov.start
require 'minitest/autorun'
require 'minitest/pride'

We'll have to update the file requirement at the top of every test to reflect this name change

**Review by AngelaGuardia — CHANGES_REQUESTED** [review]

**Comment by AngelaGuardia** [comment]
We may want to include Mathable in Game, GameTeam, and Team if the use averages

File: lib/readable.rb (hunk 1 above)
Comment (leahriffell) [reply lib/readable.rb]: Yup we can delete this readable one!

File: test/stat_tracker_test.rb [hunk 3]
```diff
@@ -1,9 +1,12 @@
-require './test/helper_test'
+require './test/test_helper'
 require './lib/stat_tracker'
+require 'mocha/minitest'
 require './lib/helpable'
+require './lib/mathable'
 
 class StatTrackerTest < Minitest::Test
   include Helpable
+  include Mathable
```
Comment (leahriffell) [inline test/stat_tracker_test.rb:10]: If the module is included in the class, is it also required in the test? I'm wondering if this also applies to Helpable

**Review by AngelaGuardia — APPROVED** [review]

**Review by hopesgit — APPROVED** [review]

**Review by leahriffell — APPROVED** [review]

**Pull request merged**